apps/ledger-live-desktop/release-notes.json

```diff
@@ -1,4 +1,12 @@
 [
```

Comment:
I don't think you need to push the release note

Reply:
Tbh don't know why it's been updated

Reply:
It's automatically added after each build of the app

Reply:
So we should revert this change everytime we build the app locally ?

Reply:
@lvndry correct, you can actually ignore this file locally with:
`git update-index --skip-worktree apps/ledger-live-desktop/release-notes.json`
and if needed undo with:
`git update-index --no-skip-worktree apps/ledger-live-desktop/release-notes.json`

It can be reverted for this PR indeed.